src/components/util/Timer.tsx

```diff
@@ -0,0 +1,46 @@
+// Timer.tsx
+
+import React, { useState, useEffect, useRef } from 'react';
+
+const Timer: React.FC = () => {
+  const [seconds, setSeconds] = useState(0);
+  const [running, setRunning] = useState(true);
+  const timerRef: any = useRef();
+
+  useEffect(() => {
+    timerRef.current = setInterval(() => {
+      if (running) setSeconds(seconds + 1);
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/taulant-eulerity/Canva/pull/4#issuecomment-2181154623>#57299f</a><span> - 06/20/2024, 05:05 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'setSeconds' function inside the 'setInterval' callback is using the 'seconds' state directly, which can lead to stale state issues because the 'seconds' value might not be the latest.
</div>
<div id="fix">
<b>Fix:</b> Use the functional form of 'setSeconds' to ensure the latest state is used.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -11,7 +11,7 @@
     timerRef.current = setInterval(() => {
 -      if (running) setSeconds(seconds + 1);
 +      if (running) setSeconds(prevSeconds => prevSeconds + 1);
     }, 1000);
```
</div>
</details>
</div>
<br>


</div>